### DISCUSSION TITLE
entering a network path in the config file for nzb download

### DISCUSSION THREAD

**Discussion by goofy84** [Tensai75/nzb-monkey-go#27] [Q&A]
hi, can anybody tell me how to enter a network path in the config file? In the last version this syntax was working \\folder1\folder2. In monkey go this isn´t working anymore. Thanks

**Comment by Tensai75** [comment]
On windows, network paths work fine if used in the correct format: `\\hostname\folder1\folder2`.
On linux just use the mounted directory: `/folder1/folder2`
